### ISSUE TITLE
T2 disguise is too early

### ISSUE THREAD

**Issue by Xymanek** [WOTCStrategyOverhaul/CovertInfiltration#294]
https://github.com/WOTCStrategyOverhaul/CovertInfiltration/blob/4da2c2f19909e58bc9175c23a04cd8b43518f80c/CovertInfiltration/Config/XComStrategyTuning.ini#L296-L297

I went mag weapons -> resistance comms. By the time mag weapons finished, I had enough corpses that all autopsies became instant, so I got T2 disguises before T1.5

Also, it grants HP, phantom and tower detection immunity - and the rest of soldiers have kelvar armor... I could just equip it on everyone to effectively get a tier of research without doing the research

TLDR: I think `AdventDisguise` needs the plated armor tech as requirement
